model/actions.py

```diff
@@ -84,9 +84,11 @@ def _kill(self, params):
         return self.action.put()
 
     def _validate_kill(self):
-        my_team = self.attacker.key.parent()
+        my_teamname = self.attacker.team
+        my_team = Team.get_by_id(my_teamanme)
```

Comment:
I believe ```get_by_id()``` returns an entity. 
You probably need to ```.id()```. 

Reference: 
https://cloud.google.com/appengine/docs/python/ndb/modelclass#Model_get_by_id
https://cloud.google.com/appengine/docs/python/ndb/keyclass#Key_id